core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/pagination/PaginationSP.java

```diff
@@ -46,66 +46,80 @@ public String printCounter() {
 
   @Override
   public String printIndex() {
-    return printIndex(null);
+    return printIndex(null, true);
   }
 
   @Override
   public String printIndex(String javascriptFunc) {
+    return printIndex(javascriptFunc, StringUtil.isNotDefined(javascriptFunc));
+  }
+
+  @Override
+  public String printIndex(String javascriptFunc, boolean nbItemsPerPage) {
     StringBuilder result = new StringBuilder();
 
-    if (getNbItems() > 0 && getNbItems() > getNbItemsPerPage()) {
+    if (getNbItems() > 5) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)